cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -23,13 +23,15 @@ module ice_dyn_shared
 
       implicit none
       private
-      public :: set_evp_parameters, stepu, step_vel, stepu_Cgrid, stepv_Cgrid, &
+      public :: set_evp_parameters, stepu, stepuCD, stepuC_E, stepuC_N, &
```

Comment:
I would call it stepvC_N.

Reply:
The location (e.g. E) is only given for the C-grid. I would rename the subroutines:
stepu becomes stepuv_B
stepuCD becomes stepuv_CD
stepuC_E becomes stepu_C
stepuC_N becomes stepv_C

Reply:
Sounds good, I can make those changes (maybe later).  I don't think we should change the names of the B subroutines yet.